src/main/webapp/app/assessment-instructions/expandable-sample-solution/expandable-sample-solution.component.html

```diff
@@ -3,6 +3,9 @@ <h5 class="expandable-header" (click)="isCollapsed = !isCollapsed">
     <fa-icon [icon]="isCollapsed ? 'angle-right' : 'angle-down'"></fa-icon>
 </h5>
 <div [ngbCollapse]="isCollapsed">
-    <jhi-modeling-editor *ngIf="sampleSolution" [readOnly]="true" [diagramType]="exercise.diagramType" [umlModel]="sampleSolution"></jhi-modeling-editor>
-    <p class="markdown-preview" [innerHTML]="formattedSampleSolutionExplanation"></p>
+    <p *ngIf="exercise.type !== ExerciseType_MODELING; else modelingInstructions" [innerHTML]="exercise.problemStatement | htmlForMarkdown"></p>
```

Comment:
Why show the problem statement as a fallback here?
e.g. text exercises have a sample solution.